### PR TITLE
Fix typo in TALM

### PR DESCRIPTION
**Pull request by sudomakeinstall2**
Signed-off-by: Saeid Askari <saskari@redhat.com>

Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by sudomakeinstall2** [comment]
@amolnar-rh

File: modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ For example, if you have a scheduled maintenance window of four hours, you can p
 You can set the timeout by configuring the `spec.remediationStrategy.timeout` setting as follows:
 [source,yaml]
 ----
-spec
+spec:
```
Comment (slovern) [inline modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc:25]: lgtm

**Comment by slovern** [comment]
@josclark42 can you give the QE ack here?
Thanks.

**Comment by josclark42** [comment]
LGTM

**Pull request closed without merging**